sdk/loadtestservice/azure-developer-loadtesting/azure/developer/loadtesting/aio/operations/_patch.py

```diff
@@ -182,6 +181,76 @@ class to perform operations on TestRun
     def __init__(self, *args, **kwargs):
         super(TestRunOperations, self).__init__(*args, **kwargs)
 
+    @overload
+    async def begin_test_run(
+        self,
+        test_run_id: str,
+        body: JSON,
+        *,
+        poll_for_test_run_status=True,
+        old_test_run_id: Optional[str] = None,
+        content_type: str = "application/merge-patch+json",
+        **kwargs: Any
+    ) -> AsyncLoadTestingLROPoller:
```

Comment:
you should include in brackets what the poller will return, i.e. `AsyncLoadTestingLROPoller[JSON]` (not sure that's what this fn actually returns, this is more of an example